pkg/client-sdk/ark_sdk.go

```diff
@@ -30,262 +34,240 @@ const (
 	DUST        = 450
 )
 
-var (
-	explorerUrlMap = map[string]string{
-		network.Liquid.Name:  "https://blockstream.info/liquid/api",
-		network.Testnet.Name: "https://blockstream.info/liquidtestnet/api",
-		network.Regtest.Name: "http://localhost:3001",
-	}
-)
-
 type ArkClient interface {
-	Connect(ctx context.Context) error
-	Balance(ctx context.Context, computeExpiryDetails bool) (*BalanceResp, error)
-	Onboard(ctx context.Context, amount uint64) (string, error)
+	Store() store.Store
```

Comment:
not sure the following is a strong enough reason:

this is used only by the wasm wrapper that exposes some ["extra methods"](https://github.com/ark-network/ark/blob/update-client-sdk/pkg/client-sdk/wasm/ark_sdk_wasm.go#L28-L36) compared to this interface for accessing asp info.

We should define the same methods here so they are consumed instead of accessing the store. Ideally, it makes much more sense to expose the wallet rather than the store but can be done in a second moment if needed.

Nice catch, I'll get rid of this in favour of getters for asp info